common/c_cpp/src/c/mempool.c

```diff
@@ -76,6 +76,7 @@ memoryNode*
 memoryPool_addNode (memoryPool* pool, size_t nodeSize)
```

Comment:
Without this change I can't compile/test on Windows (prev pull was not Windows compat). Do you want me to break this into 2 branches, one for Windows compat and the other for publisher events?